zip-0207.rst

```diff
@@ -0,0 +1,277 @@
+::
+
+  ZIP: 207
+  Title: Split Founders' Reward
+  Author: Jack Grigg <jack@z.cash>
+  Category: Consensus
+  Created: 2019-01-04
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+
+Abstract
+========
+
+This proposal defines how the consensus rules are altered to split the original Founders' Reward across three
+recipient addresses per block instead of one, corresponding to the three funding streams contained within it.
+
+
+Motivation
+==========
+
+Since the launch of the Zcash network, the consensus rules have required that until the first block reward
+halving (at block 850,000), each block must send 20% of the block subsidy to a hard-coded transparent address.
+[#block-subsidy]_ This stream of 2.5-ZEC outputs (the value after the mining slow-start was completed) can be
+split into three logical funding streams: [#continued-funding]_
+
+- (73.7%) The Founders' Reward stream
+- (14.4%) The Zcash Foundation Endowment stream
+- (11.9%) The Zerocoin Electric Coin Company Strategic Reserve stream
+
+Modifying the consensus rules to allocate the 2.5 ZEC across three separate recipient addresses decouples
+these three funding streams organizationally, legally, and operationally. It further reinforces transparency
+as to the structure of the original Founders' Reward.
+
+
+Specification
+=============
+
+Funding streams
+---------------
+
+A funding stream is defined by a constant value per block (in zatoshis), a start height (inclusive), and an
+end height (exclusive). The start height MUST be after the end of the mining slow-start (to make specification
+of the value simpler).
+
+- To-do: Decide whether to use fractional value per block instead, which gets around the mining slow-start
+  problem (and halvings) but might make implementation more complex.
+
+An active funding stream at a given block height is defined as a funding stream for which the block height is
+less than its end height, but not less than its start height.
+
+Each funding stream has an associated set of recipient addresses. Each address is used for at most 17500
+blocks, creating a roughly-monthly sequence of funding periods. The address to be used for a given block
+height is defined as follows (using ``SlowStartShift`` as-defined in [#block-subsidy]_)::
+
+    AddressChangeInterval = 17500
+    FundingStream.AddressIndex(height) =
+        floor((height - FundingStream.StartHeight) / AddressChangeInterval)
+    Address(height) = FundingStream.Addresses[FundingStream.AddressIndex(height)]
+
+- To-do: adjust ``AddressChangeInterval``, taking #3672 and #3690 (both reducing block times) into account.
```

Comment:
With the proposed change @bitcartel and @daira gave [above / in earlier review](https://github.com/zcash/zips/pull/201/files#r248786267) the *amounts* would become fractions of the total block issuance, which helps decouple from changes to block rate. Can we have the same decoupling from time?

I guess that may be messier if we try to use real-time and then map that to block heights. Is there any simple way to decouple it which still relies only on block heights directly?

If it's too messy, we can just dovetail this and the block rate proposal, but if it's possible to decouple them, that has some advantages: easier reasoning about each piece independently, we can propose changes to block rate in the future with less hassle, we can propose changes to funding streams in the future with less hassle.